build/ansible/roles/initialization/tasks/main.yml

```diff
@@ -56,7 +56,7 @@
       host: 127.0.0.1
       port: 5432
       timeout: 150
-    when: lookup('env','PMM_POSTGRES_ADDR') == '127.0.0.1:5432' and lookup('env','PMM_DISABLE_BUILTIN_POSTGRES') != '1'
+    when: lookup('env','GF_DATABASE_URL') == '' and lookup('env','GF_DATABASE_HOST') == '' and lookup('env','PMM_DISABLE_BUILTIN_POSTGRES') == ''
```

Comment:
What if the user sets `PMM_DISABLE_BUILTIN_POSTGRES` to `0`?

Reply:
it will be very weird decission 😊, but this solution is easier to support then multiple variations of True value

Reply:
Well, there is a better way, as usual. For example, we could solve it with the following syntax:

```
    when: 
      - lookup('env','GF_DATABASE_URL') == '' 
      - lookup('env','GF_DATABASE_HOST') == '' 
      - not (lookup('env','PMM_DISABLE_BUILTIN_POSTGRES') | bool)
```